src/nn/context.ts

```diff
@@ -110,4 +110,11 @@ export class MLContext {
         tf.version_core}/dist/`);
     return tf;
   }
+
+  /** @internal */
+  // Expose wasm interface for supporting configure threads for wasm backend.
+  //     wasm.setThreadsCount(n)
+  get wasm(): unknown {
```

Comment:
Thanks @BruceDai, this LGTM and please add user guide to README.md.

Reply:
> please add user guide to README.md.

Addressed it in new commit, please have another look, thanks.